backend/modules/pdf_processing/router.py

```
@@ -0,0 +1,287 @@
+from .pipeline import get_pipeline
+from .progress_service import progress_manager
+from .parser_service import parser_service
+
+logger = logging.getLogger(__name__)
+
+router = APIRouter()
+
+
+@router.post("/api/parse-pdf-fast-autofill/")
+async def parse_pdf_fast_autofill(
+    file: UploadFile = File(...),
```

Ruff (0.14.5)

Do not perform function call `File` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)